pkg/defkinds/php/builder.go

```diff
@@ -56,40 +56,84 @@ func (h *PHPHandler) WithSolver(solver statesolver.StateSolver) {
 	h.solver = solver
 }
 
-func (h *PHPHandler) Build(
-	ctx context.Context,
+func (h *PHPHandler) DebugConfig(
 	buildOpts builddef.BuildOpts,
-) (llb.State, *image.Image, error) {
-	var state llb.State
-	var img *image.Image
+) (interface{}, error) {
+	ctx := context.TODO()
```

Comment:
what's this  `TODO` method ?

Reply:
There're two ways to init a context in go: either `TODO()` or `Background()`.

For `Background()` the godoc says:
> It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.

And for `TODO()`:
> Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).

In this case, I think the code might be changed later to pass the context as argument. This would be used for instance to implement proper signal handling and context cancellation. However this is not done yet and I'm not totally sure if this would be useful. `TODO` is used here to encode that :slightly_smiling_face: